desc/objectives/_omnigenity.py

```diff
@@ -501,6 +502,232 @@ def compute(self, params, constants=None):
         return data["f_T"]
 
 
+class Omnigenity(_Objective):
+    """Omnigenity error.
```

Comment:
Would be good to explain the method and meaning of the target field etc a bit more here